src/inverseRobot.js

```diff
@@ -8,6 +8,19 @@
 
 function inverseRobot(robot) {
   // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (invertedRobot[value] !== undefined) {
+      return null;
+    }
```

Comment:
we want to check that the other way - look up [checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md)